lib/LaTeXML/Package/graphicx.sty.ltxml

```diff
@@ -43,12 +43,13 @@ DefKeyVal('Gin', 'magnifiable', '', 'true');
 
 # Redefine
 DefMacro('\includegraphics OptionalMatch:* []',
-  '\@ifnextchar[{\@includegraphics#1[#2]}{\@includegraphicx#1[#2]}');
+  '\@ifnextchar[{\@includegraphics#1[#2]}{\@includegraphicx#1[#2]}', scope => 'global');
```

Comment:
Did this sneak in from the autoload experiment? (I'm not sure that it's actually bad, here; but confusing: somehow the top-level of any binding file should always be global)

Reply:
Since we didn't merge that PR, and I still think it will be needed when graphicx gets autoloaded, I added it here as well yes.

It would seem like the top-level definitions of any binding *ought to be* global, but the default scope of DefMacro is currently local (since the "scope" option [is undefined](https://github.com/brucemiller/LaTeXML/blob/master/lib/LaTeXML/Package.pm#L1048)), so... they aren't. It's just that in regular loads, definitions rarely, if ever, get loaded from nested groups, so we don't notice.